README.md
# Fashion starter by deco.cx

A fully featured starter to help you customize and deploy an ecommerce website
in minutes.

<div style="display: flex; justify-content: center; width: 100%">
<img width="600px" height="147px"
    src="https://cdn.discordapp.com/attachments/1043241080679841793/1083140431556116553/image.png" />
</div>

## Features

This starter brings:

<ul>
  <li>Home, product and listing pages</li>
  <li>Content editable via CMS</li>
  <li>A/B test compatible</li>
  <li>Analytics</li>
  <li>Integration with VTEX, Shopify and Oracle</li>
  <li>Minicart with
    <ul>
      <li>Coupons</li>
      <li>Total</li>
      <li>Subtotal</li>
      <li>Promotions</li>
    </ul>
  </li>
  <li>SKU selector</li>
  <li>Full text search</li>
  <li>Search filters</li>
  <li>Search pagination</li>
  <li>SEO optmized
    <ul>
      <li>Meta tags</li>
      <li>Rich results</li>
      <li>OpenGraph</li>
    </ul>
  </li>
  <li>Cookie consent</li>
  <li>Carousel</li>
  <li>Product shelf</li>
  <li>SKU selector</li>
  <li>Breadcrumb</li>
  <li>Search filters</li>
  <li>Dropdown and hamburger menus</li>
  <li>Minicart</li>
  <li>Newsletter</li>
  <li>Footer</li>
  <li>Link Tree</li>
</ul>

## Getting started

To execute this website on your machine, clone it to your local machine with
`git clone` and make sure
[deno is installed](https://deno.land/manual@v1.31.1/getting_started/installation).

Then open the terminal, change the directory to where fashion was cloned and
type

```sh
deno task start
```

You should see the following output

```
$ deno task start
Task start deno run -A --watch=static/,sections/,functions/ dev.ts
Watcher Process started.
Starting live middleware: siteId=538 site=std
The manifest has been generated for 6 routes, 5 islands, 17 sections and 16 functions.
Githooks setup successfully: pre-commit
Starting live middleware: siteId=239 site=fashion
Listening on http://localhost:8000/
```

Now, open [http://localhost:8000/](http://localhost:8000/). You should see the
fashion starter running on your machine!

To continue from here, you can [perform initial setup](#initial-setup),
[understand folder structure](#understanding-folder-structure), learn how to
create [common design patterns](#recipes) (components) and comply to
[best practices](#best-practices)

## Initial setup

Below is a checklist for the store's initial setup. After completing these
tasks, your store will be ready to go live and won't need any further changes on
this repository. If, after changing the content via
[deco.cx's CMS](https://deco.cx/admin), you feel the store is not ready for
going live, you can learn how to [customize the code](#customize-the-store)

Checklist

- [ ] Ecommerce platform connected
- [ ] Fonts added
- [ ] Logo changed
- [ ] favicon.ico, robots.txt changed
- [ ] Design system changed

### Connecting your e-commerce platform

Products, prices and promotions are provided by an ecommerce platform (VTEX,
Shopify, Oracle). To connect to a platform:

Open the
[ecommerce's global config](https://deco.cx/admin/{your-site-id}/workbench?section=deco-sites%2Fstd%2Fsections%2FconfigVTEX.global.tsx)
and set your credentials. Then, open `/components/minicart/Cart.tsx` and
`/routes/api/[...catchall].tsx` and replace `bravtexfashionstore` to your
account name.

After this, your should be using the products provenient from your ecommerce
account

### Adding fonts

1. Copy & paste your font files (.woff/.woff2) to `/static/fonts` folder
2. Open `/components/GlobalTags.tsx` component and replace `@font-face` tags
   accordingly, i.e. change font-family, font-style and font-weight. Now, change
   the src attribute with the right font path.
3. Open tailwind.config.ts and change the `fontFamily` attribute accordingly.

> WARNING: Do not forget to use the `asset()` function. Not adding it will
> increase the traffic on your site and lead to higher costs

> If your don't know the font-family, font-style and font-weight, ask your
> designer for these values.

### Adding your logo

1. Make sure you have your logo in `.svg` format
2. Open your logo in your text editor and copy its content
3. Open /static/sprites.svg
4. Replace `<symbol id="Logo" .../>` with the content of your logo's `.svg`
   file. Make sure all `<svg />` tags are replaced with `<symbol />` tag and add
   the `id="Logo"` attribute. Also, make sure to have the `xmlns` attribute.

### Adding default assets

Open `static` folder and replace `favicon.ico`, `robots.txt` and
`site.webmanifest` with your own assets

### Changing style (CSS)

1. Run the store (with `deno task start`).
2. Open the
   [design system section](http://localhost:8000/_live/workbench/sections/DesignSystem.story.tsx?key=.%2Fsections%2FDesignSystem.story.tsx)
3. Now, on your text editor, open `tailwind.config.ts`
4. Change the default, header, badge, etc colors, text colors and border colors
   to better fit your desired design and check how the
   [design system section](http://localhost:8000/_live/workbench/sections/DesignSystem.story.tsx?key=.%2Fsections%2FDesignSystem.story.tsx)
   changes.

## Customize the store

If you are missing a functionality or the changes in the design system were not
enough for accomplishing your project, you can fully customize all HTML/CSS/JS
generated on your store by changing the code.

### Understanding project structure

This code uses:

1. [Deno](https://deno.land/) as runtime
2. [Preact](https://preactjs.com/) as rendering engine
3. [Fresh](https://fresh.deno.dev/) as meta framework.
4. [Deco.cx](https://www.deco.cx/) as CMS, A/B Tester and Analytics

To better encapsulate the store's code and separate it from framework specific
code, all HTML generator code was placed into `components` folder, whereas all
logic related code was placed under `sdk`. This means that UI elements, like
product shelves, footer, header, product galleries etc are all placed into
`components` whereas code for computing the prices, adding to cart etc are
placed under `sdk`. Other folders are framework-specific glue code, and serve
for:

1. `islands`: Fresh based folder for adding JavaScript to the frontend. More
   info at [Fresh's docs](https://fresh.deno.dev/docs/concepts/islands)
1. `routes`: Fresh based folder for responding custom routes to the store. More
   info at [Fresh's docs](https://fresh.deno.dev/docs/concepts/routes)
1. `static`: Fresh based folder for serving static content (assets). Check out
   [Fresh's docs](https://fresh.deno.dev/docs/concepts/static-files)
1. `import_map.json`: File containing your dependencies. Check out
   [Deno's docs](https://deno.land/manual@v1.31.0/basics/import_maps)
1. `tailwind.config.ts`: tailwindcss configuration file.
   [tailwind docs](https://tailwindcss.com/)
1. `sections`: deco.cx folder for making components both editable and composable
   on the CMS. Check out [deco.cx's docs](https://www.deco.cx/docs/en)
1. `functions`: deco.cx folder for adding dynamic data to sections provenient
   from third party APIs, like ecommerce platforms, ERPs, contentful, wordpress
   etc

### Recipes

Deco.cx's commitment to outstanding UX requires minimal work by user's devices.
This usually means shipping zero or almost to no JavaScript to browsers. This
starter uses state of the art design patterns to create rich UX with HTML and
CSS only.

Sometimes, it may be difficult to understand everything that's going on a
production-ready code like this starter's components. To learn how to create
common UI patterns with HTML and CSS only in a simpler context, take a look at
[deco.cx's recipes](https://www.deco.cx/docs/en/recipes/customizable-sections)

### Best practices

The best practices for managing the project on the long run rely around
respecting folder structure. This means:

1. Add `.tsx` files on `components` folder only
1. Add preact hooks on `sdk` folder.
1. To make a component editable, create it on the `components` folder and add an
   `export { default }` on the `sections` folder
1. To add JavaScript to the browser, create a component on the `components`
   folder and add an `export { default }` on the islands folder

Check performance best practices on [deco.cx's docs](https://www.deco.cx/docs).

# Thanks to all contributors!

<a href="https://github.com/$store/graphs/contributors">
  <img src="https://contributors-img.web.app/image?repo=deco-sites/fashion" />
</a>


CHANGELOG.md
# Changelog

All notable changes to this project will be documented in this file.

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to
[Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [Unreleased]

### Added

- [Customize Theme](https://www.notion.so/Personalizar-Theme-e8cd9479810b49a281aab9f9e39be9a8)

## Tags

### Added

### Deprecated

### Removed

### Changed

### Fixed

### Security


live.gen.ts
// DO NOT EDIT. This file is generated by deco.
// This file SHOULD be checked into source version control.
// This file is automatically updated during development when running `dev.ts`.

import { DecoManifest } from "$live/types.ts";
import * as $$$$0 from "./routes/styles.css.ts";
import * as $$$$1 from "./routes/_app.tsx";
import * as $$$$$0 from "./islands/AddToCartButton.tsx";
import * as $$$$$1 from "./islands/HeaderButton.tsx";
import * as $$$$$2 from "./islands/HeaderModals.tsx";
import * as $$$$$3 from "./islands/HeaderSearchbar.tsx";
import * as $$$$$4 from "./islands/Newsletter.tsx";
import * as $$$$$5 from "./islands/OutOfStock.tsx";
import * as $$$$$6 from "./islands/ProductImageZoom.tsx";
import * as $$$$$7 from "./islands/PromotionalBar.tsx";
import * as $$$$$8 from "./islands/SearchControls.tsx";
import * as $$$$$9 from "./islands/ShippingSimulation.tsx";
import * as $$$$$10 from "./islands/SliderJS.tsx";
import * as $$$$$11 from "./islands/WishlistButton.tsx";
import * as $$$$$$$$0 from "./sections/Category/CategoryBanner.tsx";
import * as $$$$$$$$1 from "./sections/Category/CategoryList.tsx";
import * as $$$$$$$$2 from "./sections/Content/Benefits.tsx";
import * as $$$$$$$$3 from "./sections/Content/Faq.tsx";
import * as $$$$$$$$4 from "./sections/Content/Logos.tsx";
import * as $$$$$$$$5 from "./sections/Content/Testimonials.tsx";
import * as $$$$$$$$6 from "./sections/Footer/Footer.tsx";
import * as $$$$$$$$7 from "./sections/Header/Header.tsx";
import * as $$$$$$$$8 from "./sections/Images/BannerGrid.tsx";
import * as $$$$$$$$9 from "./sections/Images/Carousel.tsx";
import * as $$$$$$$$10 from "./sections/Images/ImageGallery.tsx";
import * as $$$$$$$$11 from "./sections/Images/ShoppableBanner.tsx";
import * as $$$$$$$$12 from "./sections/Links/LinkTree.tsx";
import * as $$$$$$$$13 from "./sections/Links/Shortcuts.tsx";
import * as $$$$$$$$14 from "./sections/Miscellaneous/CampaignTimer.tsx";
import * as $$$$$$$$15 from "./sections/Miscellaneous/CookieConsent.tsx";
import * as $$$$$$$$16 from "./sections/Newsletter/Newsletter.tsx";
import * as $$$$$$$$17 from "./sections/Product/ProductDetails.tsx";
import * as $$$$$$$$18 from "./sections/Product/ProductShelf.tsx";
import * as $$$$$$$$19 from "./sections/Product/SearchResult.tsx";
import * as $$$$$$$$20 from "./sections/Product/Wishlist.tsx";
import * as $$$$$$$$21 from "./sections/Social/InstagramPosts.tsx";
import * as $$$$$$$$22 from "./sections/Social/WhatsApp.tsx";
import * as $$$$$$$$23 from "./sections/Theme/Theme.tsx";
import * as $live_workflows from "$live/routes/live/workflows/run.ts";
import * as $live_middleware from "$live/routes/_middleware.ts";
import * as $live_workbench from "$live/routes/live/workbench.ts";
import * as $live_invoke from "$live/routes/live/invoke/index.ts";
import * as $live_invoke_key from "$live/routes/live/invoke/[...key].ts";
import * as $live_editorData from "$live/routes/live/editorData.ts";
import * as $live_inspect from "$live/routes/live/inspect/[...block].ts";
import * as $live_release from "$live/routes/live/release.ts";
import * as $live_meta from "$live/routes/live/_meta.ts";
import * as $live_previews_block from "$live/routes/live/previews/[...block].tsx";
import * as $live_previews_index from "$live/routes/live/previews/index.tsx";
import * as $live_catchall from "$live/routes/[...catchall].tsx";
import * as i2$$$0 from "$live/loaders/state.ts";
import * as i2$$$1 from "$live/loaders/workflows/events.ts";
import * as i2$$$2 from "$live/loaders/workflows/get.ts";
import * as i2$$$$0 from "$live/handlers/devPage.ts";
import * as i2$$$$1 from "$live/handlers/fresh.ts";
import * as i2$$$$2 from "$live/handlers/proxy.ts";
import * as i2$$$$3 from "$live/handlers/redirect.ts";
import * as i2$$$$4 from "$live/handlers/router.ts";
import * as i2$$$$5 from "$live/handlers/routesSelection.ts";
import * as i2$$$$6 from "$live/handlers/workflowRunner.ts";
import * as i2$$$$$$$0 from "$live/pages/LivePage.tsx";
import * as i2$$$$$0 from "$live/sections/Conditional_Beta.tsx";
import * as i2$$$$$1 from "$live/sections/EmptySection.tsx";
import * as i2$$$$$2 from "$live/sections/PageInclude.tsx";
import * as i2$$$$$3 from "$live/sections/Slot.tsx";
import * as i2$$$$$4 from "$live/sections/UseSlot.tsx";
import * as i2$$$$$$$$0 from "$live/matchers/MatchAlways.ts";
import * as i2$$$$$$$$1 from "$live/matchers/MatchCron.ts";
import * as i2$$$$$$$$2 from "$live/matchers/MatchDate.ts";
import * as i2$$$$$$$$3 from "$live/matchers/MatchDevice.ts";
import * as i2$$$$$$$$4 from "$live/matchers/MatchEnvironment.ts";
import * as i2$$$$$$$$5 from "$live/matchers/MatchHost.ts";
import * as i2$$$$$$$$6 from "$live/matchers/MatchMulti.ts";
import * as i2$$$$$$$$7 from "$live/matchers/MatchRandom.ts";
import * as i2$$$$$$$$8 from "$live/matchers/MatchSite.ts";
import * as i2$$$$$$$$9 from "$live/matchers/MatchUserAgent.ts";
import * as i2$$$$$$$$$0 from "$live/flags/audience.ts";
import * as i2$$$$$$$$$1 from "$live/flags/everyone.ts";
import * as i2$$$$$$$$$2 from "$live/flags/flag.ts";
import * as i2$$$$$$$$$3 from "$live/flags/multivariate.ts";
import * as i2$$$$$$0 from "$live/actions/workflows/cancel.ts";
import * as i2$$$$$$1 from "$live/actions/workflows/signal.ts";
import * as i2$$$$$$2 from "$live/actions/workflows/start.ts";
import * as i2$0 from "deco-sites/std/functions/butterCMSAds.ts";
import * as i2$1 from "deco-sites/std/functions/butterCMSBrands.ts";
import * as i2$2 from "deco-sites/std/functions/butterCMSCategories.ts";
import * as i2$3 from "deco-sites/std/functions/butterCMSFeaturedPosts.ts";
import * as i2$4 from "deco-sites/std/functions/butterCMSPage.ts";
import * as i2$5 from "deco-sites/std/functions/butterCMSPlaces.ts";
import * as i2$6 from "deco-sites/std/functions/butterCMSPostDetail.ts";
import * as i2$7 from "deco-sites/std/functions/butterCMSPosts.ts";
import * as i2$8 from "deco-sites/std/functions/butterCMSRelatedPosts.ts";
import * as i2$9 from "deco-sites/std/functions/occProductDetailsPage.ts";
import * as i2$10 from "deco-sites/std/functions/requestToParam.ts";
import * as i2$11 from "deco-sites/std/functions/shopifyProductDetailsPage.ts";
import * as i2$12 from "deco-sites/std/functions/shopifyProductList.ts";
import * as i2$13 from "deco-sites/std/functions/shopifyProductListingPage.ts";
import * as i2$14 from "deco-sites/std/functions/vndaProductDetailsPage.ts";
import * as i2$15 from "deco-sites/std/functions/vndaProductList.ts";
import * as i2$16 from "deco-sites/std/functions/vndaProductListingPage.ts";
import * as i2$17 from "deco-sites/std/functions/vtexLegacyProductDetailsPage.ts";
import * as i2$18 from "deco-sites/std/functions/vtexLegacyProductList.ts";
import * as i2$19 from "deco-sites/std/functions/vtexLegacyProductListingPage.ts";
import * as i2$20 from "deco-sites/std/functions/vtexLegacyRelatedProductsLoader.ts";
import * as i2$21 from "deco-sites/std/functions/vtexNavbar.ts";
import * as i2$22 from "deco-sites/std/functions/vtexProductDetailsPage.ts";
import * as i2$23 from "deco-sites/std/functions/vtexProductList.ts";
import * as i2$24 from "deco-sites/std/functions/vtexProductListingPage.ts";
import * as i2$25 from "deco-sites/std/functions/vtexSuggestions.ts";
import * as i2$26 from "deco-sites/std/functions/vtexWishlist.ts";
import * as i2$$0 from "deco-sites/std/accounts/nuvemShop.ts";
import * as i2$$1 from "deco-sites/std/accounts/occ.ts";
import * as i2$$2 from "deco-sites/std/accounts/shopify.ts";
import * as i2$$3 from "deco-sites/std/accounts/vnda.ts";
import * as i2$$4 from "deco-sites/std/accounts/vtex.ts";
import * as i2$$5 from "deco-sites/std/accounts/yourViews.ts";
import * as i2$$$3 from "deco-sites/std/loaders/nuvemShop/nuvemShopProductDetailsPage.ts";
import * as i2$$$4 from "deco-sites/std/loaders/nuvemShop/nuvemShopProductList.ts";
import * as i2$$$5 from "deco-sites/std/loaders/nuvemShop/nuvemShopProductListingPage.ts";
import * as i2$$$6 from "deco-sites/std/loaders/vnda/proxy.ts";
import * as i2$$$7 from "deco-sites/std/loaders/vtex/cart.ts";
import * as i2$$$8 from "deco-sites/std/loaders/vtex/intelligentSearch/productDetailsPage.ts";
import * as i2$$$9 from "deco-sites/std/loaders/vtex/intelligentSearch/productList.ts";
import * as i2$$$10 from "deco-sites/std/loaders/vtex/intelligentSearch/productListingPage.ts";
import * as i2$$$11 from "deco-sites/std/loaders/vtex/intelligentSearch/suggestions.ts";
import * as i2$$$12 from "deco-sites/std/loaders/vtex/legacy/productDetailsPage.ts";
import * as i2$$$13 from "deco-sites/std/loaders/vtex/legacy/productList.ts";
import * as i2$$$14 from "deco-sites/std/loaders/vtex/legacy/productListingPage.ts";
import * as i2$$$15 from "deco-sites/std/loaders/vtex/legacy/relatedProductsLoader.ts";
import * as i2$$$16 from "deco-sites/std/loaders/vtex/navbar.ts";
import * as i2$$$17 from "deco-sites/std/loaders/vtex/proxy.ts";
import * as i2$$$18 from "deco-sites/std/loaders/vtex/user.ts";
import * as i2$$$19 from "deco-sites/std/loaders/vtex/wishlist.ts";
import * as i2$$$$7 from "deco-sites/std/handlers/vtex/sitemap.ts";
import * as i2$$$$$5 from "deco-sites/std/sections/Analytics.tsx";
import * as i2$$$$$6 from "deco-sites/std/sections/configButterCMS.global.tsx";
import * as i2$$$$$7 from "deco-sites/std/sections/configNuvemShop.tsx";
import * as i2$$$$$8 from "deco-sites/std/sections/configOCC.global.tsx";
import * as i2$$$$$9 from "deco-sites/std/sections/configShopify.global.tsx";
import * as i2$$$$$10 from "deco-sites/std/sections/configVTEX.global.tsx";
import * as i2$$$$$11 from "deco-sites/std/sections/configYourViews.global.tsx";
import * as i2$$$$$12 from "deco-sites/std/sections/SEO.tsx";
import * as i2$$$$$13 from "deco-sites/std/sections/SEOPDP.tsx";
import * as i2$$$$$14 from "deco-sites/std/sections/SEOPLP.tsx";
import * as i2$$$$$15 from "deco-sites/std/sections/VTEXPortalDataLayerCompatibility.tsx";
import * as i2$$$$$$3 from "deco-sites/std/actions/vtex/analytics/sendEvent.ts";
import * as i2$$$$$$4 from "deco-sites/std/actions/vtex/cart/addItems.ts";
import * as i2$$$$$$5 from "deco-sites/std/actions/vtex/cart/getInstallment.ts";
import * as i2$$$$$$6 from "deco-sites/std/actions/vtex/cart/removeItemAttachment.ts";
import * as i2$$$$$$7 from "deco-sites/std/actions/vtex/cart/removeItems.ts";
import * as i2$$$$$$8 from "deco-sites/std/actions/vtex/cart/simulation.ts";
import * as i2$$$$$$9 from "deco-sites/std/actions/vtex/cart/updateAttachment.ts";
import * as i2$$$$$$10 from "deco-sites/std/actions/vtex/cart/updateCoupons.ts";
import * as i2$$$$$$11 from "deco-sites/std/actions/vtex/cart/updateItemAttachment.ts";
import * as i2$$$$$$12 from "deco-sites/std/actions/vtex/cart/updateItemPrice.ts";
import * as i2$$$$$$13 from "deco-sites/std/actions/vtex/cart/updateItems.ts";
import * as i2$$$$$$14 from "deco-sites/std/actions/vtex/cart/updateProfile.ts";
import * as i2$$$$$$15 from "deco-sites/std/actions/vtex/cart/updateUser.ts";
import * as i2$$$$$$16 from "deco-sites/std/actions/vtex/newsletter/subscribe.ts";
import * as i2$$$$$$17 from "deco-sites/std/actions/vtex/notifyme.ts";
import * as i2$$$$$$18 from "deco-sites/std/actions/vtex/wishlist/addItem.ts";
import * as i2$$$$$$19 from "deco-sites/std/actions/vtex/wishlist/removeItem.ts";

const manifest = {
  "routes": {
    "./routes/_app.tsx": $$$$1,
    "./routes/_middleware.ts": $live_middleware,
    "./routes/[...catchall].tsx": $live_catchall,
    "./routes/index.tsx": $live_catchall,
    "./routes/live/_meta.ts": $live_meta,
    "./routes/live/editorData.ts": $live_editorData,
    "./routes/live/inspect/[...block].ts": $live_inspect,
    "./routes/live/invoke/[...key].ts": $live_invoke_key,
    "./routes/live/invoke/index.ts": $live_invoke,
    "./routes/live/previews/[...block].tsx": $live_previews_block,
    "./routes/live/previews/index.tsx": $live_previews_index,
    "./routes/live/release.ts": $live_release,
    "./routes/live/workbench.ts": $live_workbench,
    "./routes/live/workflows/run.ts": $live_workflows,
    "./routes/styles.css.ts": $$$$0,
  },
  "islands": {
    "./islands/AddToCartButton.tsx": $$$$$0,
    "./islands/HeaderButton.tsx": $$$$$1,
    "./islands/HeaderModals.tsx": $$$$$2,
    "./islands/HeaderSearchbar.tsx": $$$$$3,
    "./islands/Newsletter.tsx": $$$$$4,
    "./islands/OutOfStock.tsx": $$$$$5,
    "./islands/ProductImageZoom.tsx": $$$$$6,
    "./islands/PromotionalBar.tsx": $$$$$7,
    "./islands/SearchControls.tsx": $$$$$8,
    "./islands/ShippingSimulation.tsx": $$$$$9,
    "./islands/SliderJS.tsx": $$$$$10,
    "./islands/WishlistButton.tsx": $$$$$11,
  },
  "sections": {
    "$live/sections/Conditional_Beta.tsx": i2$$$$$0,
    "$live/sections/EmptySection.tsx": i2$$$$$1,
    "$live/sections/PageInclude.tsx": i2$$$$$2,
    "$live/sections/Slot.tsx": i2$$$$$3,
    "$live/sections/UseSlot.tsx": i2$$$$$4,
    "deco-sites/otica-isabela/sections/Category/CategoryBanner.tsx": $$$$$$$$0,
    "deco-sites/otica-isabela/sections/Category/CategoryList.tsx": $$$$$$$$1,
    "deco-sites/otica-isabela/sections/Content/Benefits.tsx": $$$$$$$$2,
    "deco-sites/otica-isabela/sections/Content/Faq.tsx": $$$$$$$$3,
    "deco-sites/otica-isabela/sections/Content/Logos.tsx": $$$$$$$$4,
    "deco-sites/otica-isabela/sections/Content/Testimonials.tsx": $$$$$$$$5,
    "deco-sites/otica-isabela/sections/Footer/Footer.tsx": $$$$$$$$6,
    "deco-sites/otica-isabela/sections/Header/Header.tsx": $$$$$$$$7,
    "deco-sites/otica-isabela/sections/Images/BannerGrid.tsx": $$$$$$$$8,
    "deco-sites/otica-isabela/sections/Images/Carousel.tsx": $$$$$$$$9,
    "deco-sites/otica-isabela/sections/Images/ImageGallery.tsx": $$$$$$$$10,
    "deco-sites/otica-isabela/sections/Images/ShoppableBanner.tsx": $$$$$$$$11,
    "deco-sites/otica-isabela/sections/Links/LinkTree.tsx": $$$$$$$$12,
    "deco-sites/otica-isabela/sections/Links/Shortcuts.tsx": $$$$$$$$13,
    "deco-sites/otica-isabela/sections/Miscellaneous/CampaignTimer.tsx":
      $$$$$$$$14,
    "deco-sites/otica-isabela/sections/Miscellaneous/CookieConsent.tsx":
      $$$$$$$$15,
    "deco-sites/otica-isabela/sections/Newsletter/Newsletter.tsx": $$$$$$$$16,
    "deco-sites/otica-isabela/sections/Product/ProductDetails.tsx": $$$$$$$$17,
    "deco-sites/otica-isabela/sections/Product/ProductShelf.tsx": $$$$$$$$18,
    "deco-sites/otica-isabela/sections/Product/SearchResult.tsx": $$$$$$$$19,
    "deco-sites/otica-isabela/sections/Product/Wishlist.tsx": $$$$$$$$20,
    "deco-sites/otica-isabela/sections/Social/InstagramPosts.tsx": $$$$$$$$21,
    "deco-sites/otica-isabela/sections/Social/WhatsApp.tsx": $$$$$$$$22,
    "deco-sites/otica-isabela/sections/Theme/Theme.tsx": $$$$$$$$23,
    "deco-sites/std/sections/Analytics.tsx": i2$$$$$5,
    "deco-sites/std/sections/configButterCMS.global.tsx": i2$$$$$6,
    "deco-sites/std/sections/configNuvemShop.tsx": i2$$$$$7,
    "deco-sites/std/sections/configOCC.global.tsx": i2$$$$$8,
    "deco-sites/std/sections/configShopify.global.tsx": i2$$$$$9,
    "deco-sites/std/sections/configVTEX.global.tsx": i2$$$$$10,
    "deco-sites/std/sections/configYourViews.global.tsx": i2$$$$$11,
    "deco-sites/std/sections/SEO.tsx": i2$$$$$12,
    "deco-sites/std/sections/SEOPDP.tsx": i2$$$$$13,
    "deco-sites/std/sections/SEOPLP.tsx": i2$$$$$14,
    "deco-sites/std/sections/VTEXPortalDataLayerCompatibility.tsx": i2$$$$$15,
  },
  "loaders": {
    "$live/loaders/state.ts": i2$$$0,
    "$live/loaders/workflows/events.ts": i2$$$1,
    "$live/loaders/workflows/get.ts": i2$$$2,
    "deco-sites/std/loaders/nuvemShop/nuvemShopProductDetailsPage.ts": i2$$$3,
    "deco-sites/std/loaders/nuvemShop/nuvemShopProductList.ts": i2$$$4,
    "deco-sites/std/loaders/nuvemShop/nuvemShopProductListingPage.ts": i2$$$5,
    "deco-sites/std/loaders/vnda/proxy.ts": i2$$$6,
    "deco-sites/std/loaders/vtex/cart.ts": i2$$$7,
    "deco-sites/std/loaders/vtex/intelligentSearch/productDetailsPage.ts":
      i2$$$8,
    "deco-sites/std/loaders/vtex/intelligentSearch/productList.ts": i2$$$9,
    "deco-sites/std/loaders/vtex/intelligentSearch/productListingPage.ts":
      i2$$$10,
    "deco-sites/std/loaders/vtex/intelligentSearch/suggestions.ts": i2$$$11,
    "deco-sites/std/loaders/vtex/legacy/productDetailsPage.ts": i2$$$12,
    "deco-sites/std/loaders/vtex/legacy/productList.ts": i2$$$13,
    "deco-sites/std/loaders/vtex/legacy/productListingPage.ts": i2$$$14,
    "deco-sites/std/loaders/vtex/legacy/relatedProductsLoader.ts": i2$$$15,
    "deco-sites/std/loaders/vtex/navbar.ts": i2$$$16,
    "deco-sites/std/loaders/vtex/proxy.ts": i2$$$17,
    "deco-sites/std/loaders/vtex/user.ts": i2$$$18,
    "deco-sites/std/loaders/vtex/wishlist.ts": i2$$$19,
  },
  "handlers": {
    "$live/handlers/devPage.ts": i2$$$$0,
    "$live/handlers/fresh.ts": i2$$$$1,
    "$live/handlers/proxy.ts": i2$$$$2,
    "$live/handlers/redirect.ts": i2$$$$3,
    "$live/handlers/router.ts": i2$$$$4,
    "$live/handlers/routesSelection.ts": i2$$$$5,
    "$live/handlers/workflowRunner.ts": i2$$$$6,
    "deco-sites/std/handlers/vtex/sitemap.ts": i2$$$$7,
  },
  "pages": {
    "$live/pages/LivePage.tsx": i2$$$$$$$0,
  },
  "matchers": {
    "$live/matchers/MatchAlways.ts": i2$$$$$$$$0,
    "$live/matchers/MatchCron.ts": i2$$$$$$$$1,
    "$live/matchers/MatchDate.ts": i2$$$$$$$$2,
    "$live/matchers/MatchDevice.ts": i2$$$$$$$$3,
    "$live/matchers/MatchEnvironment.ts": i2$$$$$$$$4,
    "$live/matchers/MatchHost.ts": i2$$$$$$$$5,
    "$live/matchers/MatchMulti.ts": i2$$$$$$$$6,
    "$live/matchers/MatchRandom.ts": i2$$$$$$$$7,
    "$live/matchers/MatchSite.ts": i2$$$$$$$$8,
    "$live/matchers/MatchUserAgent.ts": i2$$$$$$$$9,
  },
  "flags": {
    "$live/flags/audience.ts": i2$$$$$$$$$0,
    "$live/flags/everyone.ts": i2$$$$$$$$$1,
    "$live/flags/flag.ts": i2$$$$$$$$$2,
    "$live/flags/multivariate.ts": i2$$$$$$$$$3,
  },
  "actions": {
    "$live/actions/workflows/cancel.ts": i2$$$$$$0,
    "$live/actions/workflows/signal.ts": i2$$$$$$1,
    "$live/actions/workflows/start.ts": i2$$$$$$2,
    "deco-sites/std/actions/vtex/analytics/sendEvent.ts": i2$$$$$$3,
    "deco-sites/std/actions/vtex/cart/addItems.ts": i2$$$$$$4,
    "deco-sites/std/actions/vtex/cart/getInstallment.ts": i2$$$$$$5,
    "deco-sites/std/actions/vtex/cart/removeItemAttachment.ts": i2$$$$$$6,
    "deco-sites/std/actions/vtex/cart/removeItems.ts": i2$$$$$$7,
    "deco-sites/std/actions/vtex/cart/simulation.ts": i2$$$$$$8,
    "deco-sites/std/actions/vtex/cart/updateAttachment.ts": i2$$$$$$9,
    "deco-sites/std/actions/vtex/cart/updateCoupons.ts": i2$$$$$$10,
    "deco-sites/std/actions/vtex/cart/updateItemAttachment.ts": i2$$$$$$11,
    "deco-sites/std/actions/vtex/cart/updateItemPrice.ts": i2$$$$$$12,
    "deco-sites/std/actions/vtex/cart/updateItems.ts": i2$$$$$$13,
    "deco-sites/std/actions/vtex/cart/updateProfile.ts": i2$$$$$$14,
    "deco-sites/std/actions/vtex/cart/updateUser.ts": i2$$$$$$15,
    "deco-sites/std/actions/vtex/newsletter/subscribe.ts": i2$$$$$$16,
    "deco-sites/std/actions/vtex/notifyme.ts": i2$$$$$$17,
    "deco-sites/std/actions/vtex/wishlist/addItem.ts": i2$$$$$$18,
    "deco-sites/std/actions/vtex/wishlist/removeItem.ts": i2$$$$$$19,
  },
  "functions": {
    "deco-sites/std/functions/butterCMSAds.ts": i2$0,
    "deco-sites/std/functions/butterCMSBrands.ts": i2$1,
    "deco-sites/std/functions/butterCMSCategories.ts": i2$2,
    "deco-sites/std/functions/butterCMSFeaturedPosts.ts": i2$3,
    "deco-sites/std/functions/butterCMSPage.ts": i2$4,
    "deco-sites/std/functions/butterCMSPlaces.ts": i2$5,
    "deco-sites/std/functions/butterCMSPostDetail.ts": i2$6,
    "deco-sites/std/functions/butterCMSPosts.ts": i2$7,
    "deco-sites/std/functions/butterCMSRelatedPosts.ts": i2$8,
    "deco-sites/std/functions/occProductDetailsPage.ts": i2$9,
    "deco-sites/std/functions/requestToParam.ts": i2$10,
    "deco-sites/std/functions/shopifyProductDetailsPage.ts": i2$11,
    "deco-sites/std/functions/shopifyProductList.ts": i2$12,
    "deco-sites/std/functions/shopifyProductListingPage.ts": i2$13,
    "deco-sites/std/functions/vndaProductDetailsPage.ts": i2$14,
    "deco-sites/std/functions/vndaProductList.ts": i2$15,
    "deco-sites/std/functions/vndaProductListingPage.ts": i2$16,
    "deco-sites/std/functions/vtexLegacyProductDetailsPage.ts": i2$17,
    "deco-sites/std/functions/vtexLegacyProductList.ts": i2$18,
    "deco-sites/std/functions/vtexLegacyProductListingPage.ts": i2$19,
    "deco-sites/std/functions/vtexLegacyRelatedProductsLoader.ts": i2$20,
    "deco-sites/std/functions/vtexNavbar.ts": i2$21,
    "deco-sites/std/functions/vtexProductDetailsPage.ts": i2$22,
    "deco-sites/std/functions/vtexProductList.ts": i2$23,
    "deco-sites/std/functions/vtexProductListingPage.ts": i2$24,
    "deco-sites/std/functions/vtexSuggestions.ts": i2$25,
    "deco-sites/std/functions/vtexWishlist.ts": i2$26,
  },
  "accounts": {
    "deco-sites/std/accounts/nuvemShop.ts": i2$$0,
    "deco-sites/std/accounts/occ.ts": i2$$1,
    "deco-sites/std/accounts/shopify.ts": i2$$2,
    "deco-sites/std/accounts/vnda.ts": i2$$3,
    "deco-sites/std/accounts/vtex.ts": i2$$4,
    "deco-sites/std/accounts/yourViews.ts": i2$$5,
  },
  "baseUrl": import.meta.url,
};

export type Manifest = typeof manifest;

export default manifest satisfies DecoManifest;
